massa-grpc/src/public.rs

```diff
@@ -685,7 +685,7 @@ pub(crate) fn get_status(
     _request: tonic::Request<grpc_api::GetStatusRequest>,
 ) -> Result<grpc_api::GetStatusResponse, GrpcError> {
     let config = CompactConfig::default();
-    let now = MassaTime::now()?;
```

Comment:
can the error variant corresponding to this "?" be removed as well ?

Reply:
Done